atom/browser/api/atom_api_power_monitor.h

```diff
@@ -31,6 +32,11 @@ class PowerMonitor : public mate::TrackableObject<PowerMonitor>,
   void OnSuspend() override;
   void OnResume() override;
 
+  void QuerySystemIdleState(v8::Isolate* isolate,
+                            int idle_threshold,
+                            const ui::IdleCallback& callback);
+  void QuerySystemIdleTime(const ui::IdleTimeCallback& callback);
+
```

Comment:
Let's make these methods private since we're only using them with the bindings. Permissions can be loosened later as needed

Reply:
makes sense, moved it.